### ISSUE TITLE
Specs fail with Ruby 2.0.0-p0

### ISSUE THREAD

**Issue by ukd1** [titanous/yubikey#10]
```
Failures:

  1) hex decodes hex to binary
     Failure/Error: should == "i\266H\034\213\253\242\266\016\217\"\027\233X\315V"
     ArgumentError:
       invalid byte sequence in UTF-8
     # ./spec/hex_spec.rb:12:in `block (2 levels) in <top (required)>'

  2) Yubikey::Modhex decodes modhex
     Failure/Error: Yubikey::ModHex.decode('hknhfjbrjnlnldnhcujvddbikngjrtgh').should == "i\266H\034\213\253\242\266\016\217\"\027\233X\315V"
     ArgumentError:
       invalid byte sequence in UTF-8
     # ./spec/yubikey/modhex_spec.rb:4:in `block (2 levels) in <top (required)>'
```

**Comment by ukd1** [comment]
@maartenvg awesome. I've released 1.3 now.

**Comment by titanous** [comment]
@ukd1 Great! Make sure you add a git tag `v1.3.0` with the commit that you released.
